platsec.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+ARTIFACTS_DIR="grype-vuln-artifacts"
+IMAGE_TAG="quay.io/gburges/kibana:latest"
+
+mkdir -p "$ARTIFACTS_DIR"
+
+#install and run syft
+curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b ./bin
+./bin/syft -v ${IMAGE_TAG} > "${ARTIFACTS_DIR}/syft-sbom-results.txt"
+
+#install and run grype
+curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b ./bin
+./bin/grype -v -o table ${IMAGE_TAG} > "${ARTIFACTS_DIR}/grype-vuln-results-full.txt"
+./bin/grype -v -o table --only-fixed --fail-on high ${IMAGE_TAG} > "${ARTIFACTS_DIR}/grype-vuln-results-fixable.txt"
```

Comment:
wrap this in a `if` condition to fail with a human legible error - such as "security scan failed" or similar, then exit 1.

Reply:
so turns out we actually have to run it 3 times because it doesn't create the file if it fails. So either rework the grype and cat the output for high to trigger the fail, or run it another time

Reply:
Can you please elaborate ? I've just run it with a lower threshold, and I see the output too , with an additional error message : 

```
1 error occurred:
	* discovered vulnerabilities at or above the severity threshold
```

Is this not what we expected to happen ?

Reply:
yeahm, i commented this before i added the exit 1, so now it's moot